packages/api/db/tables.sql

```diff
@@ -87,11 +89,14 @@ CREATE TABLE IF NOT EXISTS public.user_tag
   tag             user_tag_type                                                 NOT NULL,
   value           TEXT                                                          NOT NULL,
   value_type      user_tag_value_type                                           NOT NULL,
+  reason          TEXT                                                          NOT NULL,
   inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now())     NOT NULL,
   deleted_at  TIMESTAMP WITH TIME ZONE
 );
-
-CREATE INDEX IF NOT EXISTS user_updated_at_idx ON public.user (updated_at);
+CREATE UNIQUE INDEX user_tag_is_deleted_idx ON user_tag (user_id, tag, deleted_at)
```

Comment:
```suggestion
CREATE UNIQUE INDEX IF NOT EXISTS user_tag_is_deleted_idx ON user_tag (user_id, tag, deleted_at)
```

Otherwise multiple runs will create issues